README.md

```diff
@@ -1,20 +1,13 @@
-Require-to-publish
-==================
+# Require-to-publish
```

Comment:
Is this actually the human name of this module? With hyphens in it? The .info file should have an official human readable name